lib/analytics.js

```diff
@@ -603,9 +668,12 @@ Analytics.prototype._callback = function(fn) {
  * @api private
  */
 
-Analytics.prototype._invoke = function(method, facade) {
+Analytics.prototype._invoke = function(method, facade, skipEmit) {
```

Comment:
May be a good idea to document why we called this `skipEmit` and not `emit`.

Reply:
Backwards compatibility, no? This way it's not changing the default behavior of the func?

Reply:
yep! just saying we should document it why so it's obvious to someone reading this a few months later.

Reply:
Done